### PR TITLE
Feature/edit evidence note

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
More tests to come...
Edit View Model map tests
EditNoteMapTransfer tests
ModifyNoteTransfer tests
Edit Request creator test
Move success message to model mapper and tests
Controller EditEvidenceNote POST tests

### CODE REVIEW

File: src/EA.Weee.Integration.Tests/Builders/EvidenceNoteDbSetup.cs [hunk 1]
```diff
@@ -63,5 +63,13 @@ public EvidenceNoteDbSetup WithTonnages(List<NoteTonnage> tonnages)
             DefaultNote(tonnages);
             return this;
         }
+
+        //public EvidenceNoteDbSetup UpdateTonnages(List<NoteTonnage> tonnages)
```
Comment (NateHogan) [inline src/EA.Weee.Integration.Tests/Builders/EvidenceNoteDbSetup.cs]: Is this ever needed again? Would it be neater to just remove?

Comment (JohnPadgettCivica) [reply src/EA.Weee.Integration.Tests/Builders/EvidenceNoteDbSetup.cs]: Should be gone now...

File: src/EA.Weee.Web/ViewModels/Shared/Utilities/AddressUtilities.cs [hunk 2]
```diff
@@ -56,5 +56,42 @@ public string FormattedAddress(AddressData address, bool includeSiteName = true)
...
+            {
+                siteAddressLong += $"<br/>{county}";
+            }
+
+            if (postCode != null)
+            {
+                siteAddressLong += $"<br/>{postCode}";
+            }
+
+            return siteAddressLong;
```
Comment (NateHogan) [inline src/EA.Weee.Web/ViewModels/Shared/Utilities/AddressUtilities.cs:94]: Might be neater and efficient to do a StringBuilder here?

**Review by GuidoWorkDev — APPROVED** [review]
all looking good, John.

**Pull request merged**